test/src/sdk-harness/test_projects/tx_fields/mod.rs

```diff
@@ -618,22 +607,37 @@ mod inputs {
         }
 
         mod message {
-            use fuel_vm::{checked_transaction::EstimatePredicates, prelude::Cacheable};
+            use fuels::types::transaction_builders::TransactionBuilder;
 
             use super::*;
 
             #[tokio::test]
-            async fn can_get_input_message_sender() -> Result<()> {
+            async fn can_get_input_message_sender() {
                 let (contract_instance, _, wallet, _) = get_contracts().await;
-                let handler = contract_instance.methods().get_input_message_sender(2);
-                let mut tx = handler.build_tx().await.unwrap();
-                add_message_input(&mut tx, wallet.clone()).await; // let result = contract_instance
-                tx.tx.precompute(&ChainId::default())?;
 
-                let messages = wallet.get_messages().await?;
+                let message = &wallet.get_messages().await.unwrap()[0];
+                let mut builder = contract_instance.methods()
+                    .get_input_message_sender(3)
+                    .transaction_builder()
+                    .await
+                    .unwrap();
+
+                wallet.adjust_for_fee(&mut builder, 1000).await.unwrap();
```

Comment:
You could pay with a message without data and not have to `adjust_for_fee`

Reply:
Done.